README.md

```diff
@@ -273,6 +299,16 @@ myUI("sub", "overlay-hide", function(eventName) {
 });
 ```
 
+#### Autocomplete Selected
+
+An autocomplete selected event is triggered when a user presses enter while highlighting an autocomplete suggestion or clicks on an autocomplete suggestion.
```

Comment:
...to trigger a search using the suggestion.

Reply:
Is this now `suggestion-chosen`?